### ISSUE TITLE
Bike pumps: displaying amenity=compressed_air ?

### ISSUE THREAD

**Issue by gonzen** [pietervdvn/MapComplete#1541]
Hi, on CycloFix, are public pumps mapped as amenity=compressed_air also displayed ?
I cannot see them, and see only the amenity=bicycle_repair_station with service:bicycle:pump.

any idea how this could be fixed ?
best regards,
Martin

**Comment by pietervdvn** [comment]
No, only `bicycle repair stations` are shown. This was a deliberate choice made when cyclofix was setup. @joostschouppe Do you perhaps still know the reason? Was it written down somewhere?

IIRC it had to do that it was often unclear if compressed air was also available for cyclists...

**Comment by pietervdvn** [comment]
I did a small research, and `amenity=compressed_air` is getting disused more and more for bicycle pumps. I've clarified this in the wiki and retagged a few of them.

Hence, no, this will not be implemented in cyclofix.

If someone were to make a layer specifically about other compressed air amenities, I'd gladly merge it in though.

**Comment by Discostu36** [comment]
> I did a small research, and `amenity=compressed_air` is getting disused more and more for bicycle pumps. I've clarified this in the wiki and retagged a few of them.

Why is it getting disused? How do you retag it?

**Comment by pietervdvn** [comment]
I've described this on the wiki page for compressed air: https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dcompressed_air

**Comment by Discostu36** [comment]
I disagree with this tagging practice. Tagging a place as a bicycle repair station where you can't repair your bicycle is close to a troll tag for me. An air pump is an air pump and should be tagged as such.

**Comment by pietervdvn** [comment]
Pleasr, have this discussion on the wiki and on the forum. Once a new consensus is reached, I'll gladly modify the tagging.
